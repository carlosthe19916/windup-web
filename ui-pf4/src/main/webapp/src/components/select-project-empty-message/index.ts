export { SelectProjectEmptyMessage } from "./select-project-empty-message";
