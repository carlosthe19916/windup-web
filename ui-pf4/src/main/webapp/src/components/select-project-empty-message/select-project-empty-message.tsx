import React from "react";
import {
  EmptyState,
  EmptyStateIcon,
  Title,
  EmptyStateBody,
  EmptyStateVariant,
  Bullseye,
} from "@patternfly/react-core";
import { MiddlewareIcon } from "@patternfly/react-icons";

export interface SelectProjectEmptyMessageProps {}

export const SelectProjectEmptyMessage: React.FC<SelectProjectEmptyMessageProps> = () => {
  return (
    <Bullseye>
      <EmptyState variant={EmptyStateVariant.small}>
        <EmptyStateIcon icon={MiddlewareIcon} />
        <Title headingLevel="h4" size="lg">
          Select a Project
        </Title>
        <EmptyStateBody>
          Please select the project you want to see data from.
        </EmptyStateBody>
      </EmptyState>
    </Bullseye>
  );
};
