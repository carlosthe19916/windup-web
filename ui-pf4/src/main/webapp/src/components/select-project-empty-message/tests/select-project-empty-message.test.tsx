import React from "react";
import { shallow } from "enzyme";
import { SelectProjectEmptyMessage } from "../select-project-empty-message";

describe("SelectProjectEmptyMessage", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<SelectProjectEmptyMessage />);
    expect(wrapper).toMatchSnapshot();
  });
});
