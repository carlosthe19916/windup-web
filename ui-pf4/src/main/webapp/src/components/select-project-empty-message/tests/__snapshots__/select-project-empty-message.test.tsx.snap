// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectProjectEmptyMessage Renders without crashing 1`] = `
<Bullseye>
  <EmptyState
    variant="small"
  >
    <EmptyStateIcon
      icon={[Function]}
    />
    <Title
      headingLevel="h4"
      size="lg"
    >
      Select a Project
    </Title>
    <EmptyStateBody>
      Please select the project you want to see data from.
    </EmptyStateBody>
  </EmptyState>
</Bullseye>
`;
